easybuild/base/generaloption.py

```diff
@@ -103,6 +103,11 @@ def what_str_list_tuple(name):
         sep = os.pathsep
         helpsep = 'pathsep'
 
```

Comment:
Can you remove this empty line please? And as we have 3 cases now, I think it would be great to put the comma part above into an `else` clause below this (again without empty line between previous "helpset" and "else")